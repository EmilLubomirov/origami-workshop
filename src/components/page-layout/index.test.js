import React from "react";
import renderer from "react-test-renderer";
import TestingEnvironment from "../../test-utils/router";
import PageLayout from "./index";
jest.mock('../header/', () => 'Header');
jest.mock('../aside/', () => 'Aside');
jest.mock('../footer/', () => 'Footer');

describe('Page Layout Component', function () {

    it('should render page layout component', function () {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    id: 123
                },
                isLoggedIn: true
            }}>
                <PageLayout/>
            </TestingEnvironment>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});