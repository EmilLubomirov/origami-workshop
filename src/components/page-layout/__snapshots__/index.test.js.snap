// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Layout Component should render page layout component 1`] = `
Array [
  <Header />,
  <Aside />,
  <main
    className="main"
  />,
  <Footer />,
]
`;
